factory/pom.xml

```diff
@@ -83,12 +83,6 @@
       <artifactId>auto-service-annotations</artifactId>
       <version>${auto-service.version}</version>
     </dependency>
-    <dependency>
-      <groupId>net.ltgt.gradle.incap</groupId>
-      <artifactId>incap</artifactId>
-      <version>0.3</version>
-      <scope>provided</scope>
-    </dependency>
```

Comment:
This is the annotation and enum, it should stay here; the incap-processor needs to be moved.